osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -33,8 +33,8 @@ public class ChannelManager : PollingComponent, IChannelPostTarget
         private readonly BindableList<Channel> availableChannels = new BindableList<Channel>();
         private readonly BindableList<Channel> joinedChannels = new BindableList<Channel>();
 
-        // Keeps a list of closed channels. More recently closed channels appear at higher indeces
-        private readonly BindableList<Channel> closedChannels = new BindableList<Channel>();
+        // Keeps a list of closed channel identifiers
+        private readonly BindableList<string> closedChannels = new BindableList<string>();
 
         // For efficiency purposes, this constant bounds the number of closed channels we store.
         // This number is somewhat arbitrary; future developers are free to modify it.
```

Comment:
I think bdach also left another reply on the max size: [https://github.com/ppy/osu/pull/11182#discussion_r544165092](https://github.com/ppy/osu/pull/11182#discussion_r544165092)